File: wingc/test-files/nested.py
from wing_rpc import Schema, Enum
from typing import ClassVar
from enum import StrEnum


class Address(Schema):
    pass


class Person(Schema):
    __match_args__: ClassVar[tuple] = ('name', 'age', 'address',)
    name: str
    age: int
    address: Address

File: wingc/test-files/nested.rs
use serde::{Serialize, Deserialize};
use wing_rpc::Message as WingMessage;


#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
pub struct Address {
}

impl<'a> WingMessage<'a> for Address {
    const NAME: &'static str = "Address";
}

#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
pub struct Person {
    pub name: String,
    pub age: u8,
    pub address: Address,
}

impl<'a> WingMessage<'a> for Person {
    const NAME: &'static str = "Person";
}

